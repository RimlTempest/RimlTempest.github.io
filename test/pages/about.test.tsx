import React from 'react';
import { render } from '../testUtils';
import About from '../../src/pages/about/index';

describe('Aboutページのテスト', () => {
  describe('スナップショットと合致するか', () => {
    test('snapshot', () => {
      const { asFragment } = render(<About />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
