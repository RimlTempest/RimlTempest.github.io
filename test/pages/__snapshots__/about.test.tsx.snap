// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aboutページのテスト スナップショットと合致するか snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1k8qnsz"
  >
    <div
      class="css-1qli9sf"
    >
      <div
        class="css-jfo7z"
      >
        <button
          aria-label="Open Menu"
          class="chakra-button css-cb9edx"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="chakra-icon css-onkibi"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M 3 5 A 1.0001 1.0001 0 1 0 3 7 L 21 7 A 1.0001 1.0001 0 1 0 21 5 L 3 5 z M 3 11 A 1.0001 1.0001 0 1 0 3 13 L 21 13 A 1.0001 1.0001 0 1 0 21 11 L 3 11 z M 3 17 A 1.0001 1.0001 0 1 0 3 19 L 21 19 A 1.0001 1.0001 0 1 0 21 17 L 3 17 z"
              fill="currentColor"
            />
          </svg>
        </button>
        <div
          class="chakra-stack css-1ukngxm"
        >
          <nav
            class="chakra-stack css-dss5yb"
          >
            <a
              class="chakra-link css-1cdzvrw"
            >
              トップ
            </a>
            <a
              class="chakra-link css-1cdzvrw"
            >
              私について
            </a>
            <a
              class="chakra-link css-1cdzvrw"
            >
              成果物
            </a>
            <a
              class="chakra-link css-1cdzvrw"
            >
              連絡先
            </a>
          </nav>
        </div>
        <div
          class="css-70qvj9"
        >
          <div
            class="css-1hyoz7m"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M283.211 512c78.962 0 151.079-35.925 198.857-94.792 7.068-8.708-.639-21.43-11.562-19.35-124.203 23.654-238.262-71.576-238.262-196.954 0-72.222 38.662-138.635 101.498-174.394 9.686-5.512 7.25-20.197-3.756-22.23A258.156 258.156 0 0 0 283.211 0c-141.309 0-256 114.511-256 256 0 141.309 114.511 256 256 256z"
              />
            </svg>
            <label
              class="chakra-switch css-uny7qy"
            >
              <input
                class="chakra-switch__input"
                style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
                type="checkbox"
                value=""
              />
              <span
                aria-hidden="true"
                class="chakra-switch__track css-14qxnv8"
              >
                <span
                  class="chakra-switch__thumb css-0"
                />
              </span>
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      class="css-uwwqev"
      style="opacity: 0;"
    >
      <div
        class="css-1fqpna9"
      >
        <div
          class="css-189g9dp"
        >
          <div
            class="css-1kzgbw3"
          >
            <h2
              class="chakra-heading css-1xpj11g"
            >
              私について
            </h2>
          </div>
          <div
            class="css-16r9zrd"
          >
            <div
              class="css-y9adcs"
            >
              <div
                class="css-1dorgyd"
              />
              <div
                class="css-2svpkz"
              >
                <span
                  class="chakra-avatar css-1ruhx4w"
                >
                  <svg
                    class="css-16ite8i"
                    role="img"
                    viewBox="0 0 128 128"
                  >
                    <path
                      d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                      fill="currentColor"
                    />
                    <path
                      d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <div
                  class="css-1h3nzko"
                >
                  <p
                    class="chakra-text css-14vi7by"
                  >
                    Riml
                  </p>
                  <p
                    class="chakra-text css-g5hezf"
                  >
                    髙橋 大貴
                  </p>
                </div>
              </div>
              <div
                class="css-j7qwjs"
              >
                <div
                  class="css-18f4n7l"
                >
                  <p
                    class="chakra-text css-0"
                  >
                    趣味
                  </p>
                  <div
                    class="css-13vecxt"
                  >
                    <ul
                      class="css-49duqd"
                      role="list"
                    >
                      <div
                        class="chakra-accordion css-0"
                      >
                        <div
                          class="chakra-accordion__item css-1fsnuue"
                        >
                          <li
                            class="css-0"
                          >
                            <button
                              aria-controls="accordion-panel-2"
                              aria-expanded="false"
                              class="chakra-accordion__button css-9179w8"
                              id="accordion-button-2"
                              type="button"
                            >
                              <div
                                class="css-v6s5ft"
                              >
                                <p
                                  class="chakra-text css-0"
                                >
                                  VTuber、アニメ鑑賞などの動画鑑賞
                                </p>
                              </div>
                              <svg
                                aria-hidden="true"
                                class="chakra-icon css-3bodus"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                  fill="currentColor"
                                />
                              </svg>
                            </button>
                            <div
                              class="chakra-collapse"
                              style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                            >
                              <div
                                aria-labelledby="accordion-button-2"
                                class="chakra-accordion__panel css-b0qdn7"
                                id="accordion-panel-2"
                                role="region"
                              >
                                <div
                                  class="css-u4p24i"
                                >
                                  <p
                                    class="chakra-text css-g5hezf"
                                  >
                                    VTuber最推しはホロライブ三期生の
                                  </p>
                                  <span
                                    class="chakra-badge css-8jwegb"
                                  >
                                    潤羽るしあ
                                  </span>
                                </div>
                                <div
                                  class="css-u4p24i"
                                >
                                  <p
                                    class="chakra-text css-g5hezf"
                                  >
                                    一番好きなアニメは
                                  </p>
                                  <span
                                    class="chakra-badge css-8jwegb"
                                  >
                                    転生したらスライムだった件
                                  </span>
                                </div>
                              </div>
                            </div>
                          </li>
                        </div>
                        <div
                          class="chakra-accordion__item css-1fsnuue"
                        >
                          <li
                            class="css-0"
                          >
                            <button
                              aria-controls="accordion-panel-3"
                              aria-expanded="false"
                              class="chakra-accordion__button css-9179w8"
                              id="accordion-button-3"
                              type="button"
                            >
                              <div
                                class="css-v6s5ft"
                              >
                                <p
                                  class="chakra-text css-0"
                                >
                                  リズムゲーム
                                </p>
                              </div>
                              <svg
                                aria-hidden="true"
                                class="chakra-icon css-3bodus"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                  fill="currentColor"
                                />
                              </svg>
                            </button>
                            <div
                              class="chakra-collapse"
                              style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                            >
                              <div
                                aria-labelledby="accordion-button-3"
                                class="chakra-accordion__panel css-b0qdn7"
                                id="accordion-panel-3"
                                role="region"
                              >
                                <div
                                  class="css-u4p24i"
                                >
                                  <span
                                    class="chakra-badge css-ifsvay"
                                  >
                                    第2回ガルパ杯
                                  </span>
                                  <p
                                    class="chakra-text css-11a6vfm"
                                  >
                                    二次予選出場（仙台）
                                  </p>
                                </div>
                                <div
                                  class="css-u4p24i"
                                >
                                  <span
                                    class="chakra-badge css-ifsvay"
                                  >
                                    チュウニズム
                                  </span>
                                  <p
                                    class="chakra-text css-g5hezf"
                                  >
                                    、
                                  </p>
                                  <span
                                    class="chakra-badge css-ifsvay"
                                  >
                                    maimai
                                  </span>
                                  <p
                                    class="chakra-text css-11a6vfm"
                                  >
                                    虹レート到達
                                  </p>
                                </div>
                                <div
                                  class="css-u4p24i"
                                >
                                  <span
                                    class="chakra-badge css-ifsvay"
                                  >
                                    BeatTube
                                  </span>
                                  <p
                                    class="chakra-text css-11a6vfm"
                                  >
                                    プレイヤーランキング 最大５位
                                  </p>
                                </div>
                              </div>
                            </div>
                          </li>
                        </div>
                        <div
                          class="chakra-accordion__item css-1fsnuue"
                        >
                          <li
                            class="css-0"
                          >
                            <button
                              aria-controls="accordion-panel-4"
                              aria-expanded="false"
                              class="chakra-accordion__button css-9179w8"
                              id="accordion-button-4"
                              type="button"
                            >
                              <div
                                class="css-v6s5ft"
                              >
                                <p
                                  class="chakra-text css-0"
                                >
                                  プログラミング
                                </p>
                              </div>
                              <svg
                                aria-hidden="true"
                                class="chakra-icon css-3bodus"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                  fill="currentColor"
                                />
                              </svg>
                            </button>
                            <div
                              class="chakra-collapse"
                              style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                            >
                              <div
                                aria-labelledby="accordion-button-4"
                                class="chakra-accordion__panel css-b0qdn7"
                                id="accordion-panel-4"
                                role="region"
                              >
                                <p
                                  class="chakra-text css-g5hezf"
                                >
                                  Connect2019　優秀賞
                                </p>
                                <p
                                  class="chakra-text css-g5hezf"
                                >
                                  校内ハッカソン最優秀賞、企業賞
                                </p>
                              </div>
                            </div>
                          </li>
                        </div>
                      </div>
                    </ul>
                  </div>
                </div>
                <div
                  class="css-18i4rnw"
                >
                  <p
                    class="chakra-text css-o7rt0f"
                  >
                    スキル
                  </p>
                  <div
                    class="css-ckum9g"
                  >
                    <div
                      class="chakra-tabs css-8atqhb"
                    >
                      <div
                        aria-orientation="horizontal"
                        class="chakra-tabs__tablist css-1qol2bl"
                        role="tablist"
                      >
                        <button
                          aria-controls="tabs-5--tabpanel-0"
                          aria-selected="true"
                          class="chakra-tabs__tab css-171dlbr"
                          id="tabs-5--tab-0"
                          role="tab"
                          tabindex="0"
                          type="button"
                        >
                          プログラミング
                        </button>
                        <button
                          aria-controls="tabs-5--tabpanel-1"
                          aria-selected="false"
                          class="chakra-tabs__tab css-171dlbr"
                          id="tabs-5--tab-1"
                          role="tab"
                          tabindex="-1"
                          type="button"
                        >
                          フレームワーク
                        </button>
                      </div>
                      <div
                        class="chakra-tabs__tab-panels css-8atqhb"
                      >
                        <div
                          aria-labelledby="tabs-5--tab-0"
                          class="chakra-tabs__tab-panel css-1jj07fx"
                          id="tabs-5--tabpanel-0"
                          role="tabpanel"
                          tabindex="0"
                        >
                          <table
                            class="chakra-table css-0"
                            role="table"
                          >
                            <thead
                              class="css-0"
                            >
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <th
                                  class="css-11jcb1p"
                                >
                                  プログラミング
                                </th>
                                <th
                                  class="css-xi606m"
                                >
                                  学習率
                                </th>
                              </tr>
                            </thead>
                            <tbody
                              class="css-0"
                            >
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  HTML
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="49"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 49%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  SCSS
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="53"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 53%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  JavaScript
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="66"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 66%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  TypeScript
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="66"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 66%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                            </tbody>
                          </table>
                        </div>
                        <div
                          aria-labelledby="tabs-5--tab-1"
                          class="chakra-tabs__tab-panel css-1jj07fx"
                          hidden=""
                          id="tabs-5--tabpanel-1"
                          role="tabpanel"
                          tabindex="0"
                        >
                          <table
                            class="chakra-table css-0"
                            role="table"
                          >
                            <thead
                              class="css-0"
                            >
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <th
                                  class="css-11jcb1p"
                                >
                                  フレームワーク
                                </th>
                                <th
                                  class="css-xi606m"
                                >
                                  学習率
                                </th>
                              </tr>
                            </thead>
                            <tbody
                              class="css-0"
                            >
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  Next.js
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="68"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 68%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  React.js
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="68"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 68%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  Nuxt.js
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="57"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 57%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                              <tr
                                class="css-0"
                                role="row"
                              >
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  Vue.js
                                </td>
                                <td
                                  class="css-0"
                                  role="gridcell"
                                >
                                  <div
                                    class="css-jnrlws"
                                  >
                                    <div
                                      aria-valuemax="100"
                                      aria-valuemin="0"
                                      aria-valuenow="55"
                                      class="css-geebuc"
                                      role="progressbar"
                                      style="width: 55%;"
                                    />
                                  </div>
                                </td>
                              </tr>
                            </tbody>
                          </table>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="css-1184bef"
    >
      <div
        class="chakra-stack chakra-container css-1ugegxx"
      >
        <p
          class="chakra-text css-0"
        >
          © 2021 Daiki Takahashi. All rights reserved
        </p>
        <div
          class="chakra-stack css-57svvl"
        >
          <a
            class="css-r5t8r3"
            href="https://twitter.com/Fande4d"
          >
            <span
              class="css-f8n5zr"
            >
              Twitter
            </span>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
          <a
            class="css-r5t8r3"
            href="https://github.com/RimlTempest"
          >
            <span
              class="css-f8n5zr"
            >
              Github
            </span>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 496 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
